### DISCUSSION TITLE
Can I build for iOS target?

### DISCUSSION THREAD

**Discussion by chocoford** [sparkle-project/Sparkle#2330] [Q&A]
### Description of the problem
I am trying to build a multi-platform app and have used the Sparkle framework in the project for the macOS platform. However, Xcode is giving me the error While building for iOS, no library for this platform was found in '.../Sparkle.xcframework', even though I have marked Sparkle as for macOS in the Frameworks, Libraries, and Embedded Content section.

### Do you use Sandboxing in your app?
Yes

### Version of `Sparkle.framework` in the latest version of your app
`2.3.1`

### Steps to reproduce the behavior
Just create a Multiplatform App with Xcode. Add Sparkle package. Mark `Sparkle` as macOS only. Then build for iOS device.
![image](https://user-images.githubusercontent.com/28218759/221527355-fa47391d-e86c-4e24-a347-54a64de246be.png)

**Comment by zorgiepoo** [comment]
This might be an Xcode + Swift Package Manager bug that may be worth filing to Apple via Feedback Assistant. I don't think the issue occurs if you copy the (xc)framework to your project when not using SPM. Other workarounds are having separate targets for iOS and macOS.

**Comment by andreasley** [comment]
This seems to have been resolved in Xcode 15.

I was able to create a Multiplatform App (macOS and iOS), add the `Sparkle` 2.5.0 package , mark the Framework as "macOS" in the Target and build/run the app successfully on both macOS 13.5.2 and iOS 17.
